stdlib/http/src/main/ballerina/src/http/http_request.bal

```diff
@@ -534,7 +535,14 @@ public type Request object {
     # + cookiesToAdd - Represents the cookies to be added
     public function addCookies(Cookie[] cookiesToAdd) {
         string cookieheader = "";
-        Cookie[] sortedCookies = cookiesToAdd.sort(comparator);
+        Cookie[] sortedCookies = cookiesToAdd.sort(array:ASCENDING, function(Cookie c) returns int {
+            var cookiePath = c.path;
+            int l = 0;
+            if (cookiePath is string) {
+                l = cookiePath.length();
+            }
+            return l;
+        });
```

Comment:
This changes the original ordering behaviour implemented by the provided comparator. I think we'll have to sort it twice to achieve the same effect: first, sort using the length as above and second, sort using the created time. @chamil321 thoughts?

Reply:
Yes. aware of it. This was communicated with @daneshk and I was asked to create an issue in the HTTP module repo and fix it here just to get the PR checks passed.

Reply:
Ah ok.